Kinvey/Kinvey/Realtime.swift

```diff
@@ -504,11 +504,11 @@ extension LiveStreamAclGroups: StaticMappable {
  */
 extension LiveStream: Hashable {
     
-    public var hashValue: Int {
+    internal var hashValue: Int {
         return uuid.hashValue
     }
     
-    public static func ==(lhs: LiveStream<Type>, rhs: LiveStream<Type>) -> Bool {
+    internal static func ==(lhs: LiveStream<Type>, rhs: LiveStream<Type>) -> Bool {
         return lhs.uuid == rhs.uuid
     }
     
```

Comment:
Since some of these deprecated methods are now just internal, should we just delete them?

Reply:
sure!